NewHorizons/Handlers/HeldItemHandler.cs

```diff
@@ -0,0 +1,229 @@
+using HarmonyLib;
+using NewHorizons.Builder.Props;
+using NewHorizons.External.Modules.Props;
+using NewHorizons.Utility;
+using NewHorizons.Utility.OWML;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace NewHorizons.Handlers;
+
+[HarmonyPatch]
+public static class HeldItemHandler
```

Comment:
can we get a comment somewhere at the top of this file saying how goofy and cursed this is